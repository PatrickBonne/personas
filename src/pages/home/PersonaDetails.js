
import React from 'react'

class PersonaDetails extends React.Component {
  render () {
    return (
      <div>
        <h1>Persona details</h1>
        <p>
          { this.props.persona }
        </p>
      </div>
    )
  }
}

export { PersonaDetails }
