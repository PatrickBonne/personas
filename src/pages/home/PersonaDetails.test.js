import React from 'react'
import renderer from 'react-test-renderer'
import { PersonaDetails } from './PersonaDetails'

it('renders correctly', () => {
  const tree = renderer
    .create(<PersonaDetails persona='Chuck' />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
