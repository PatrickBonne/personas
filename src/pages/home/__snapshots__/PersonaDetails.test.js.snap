// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Persona details
  </h1>
  <p>
    Chuck
  </p>
</div>
`;
